samples/powercli/binding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sources.tanzu.vmware.com/v1alpha1
+kind: VSphereBinding
+metadata:
+  name: vsphere-jobs
+spec:
+  subject:
+    apiVersion: batch/v1
+    kind: Job
+    selector:
+      matchLabels:
+        role: vsphere-job
+
+  address: REPLACE_ME
```

Comment:
do you want a comment here also about needing to point to real Vsphere? Could use a yaml comment?

Reply:
done